django/src/chat/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
remove comment

Reply:
auto-generated by djagno, it's ok to leave it